website/versioned_docs/version-0.60/sectionlist.md

```diff
@@ -380,9 +380,9 @@ Rendered at the top and bottom of each section (note this is different from `Ite
 
 Makes section headers stick to the top of the screen until the next one pushes it off. Only enabled by default on iOS because that is the platform standard there.
 
-| Type    | Required |
-| ------- | -------- |
-| boolean | No       |
+| Type    | Required                                                                                         |
```

Comment:
@gabrieldonadel Thank you for the review updates! 

```suggestion
| Type    | Default                                                                                          |
```

Between the versions there were some changes to value tables. Please update the header as suggested above, here and in few other files. Other than that changes LGTM! 👍

Reply:
@Simek fixed 🙂